domain/authentication/login.go

```diff
@@ -21,24 +22,69 @@ func (auc UseCase) AuthenticatedLogin(cpf, secret string) (error, string) {
 	}
 
 	newLogin := domain.Login{CPF: cpf, Secret: secretHash}
-	account := auc.AccountUseCase.GetAccountCPF(cpf)
+	account, err := auc.AccountUseCase.GetAccountCPF(cpf)
+
+	if err != nil {
+		return domain.ErrInsert, ""
+	}
 
 	err = CheckLogin(account, newLogin)
 	if err != nil {
 		return domain.ErrLogin, ""
 	}
 
-	id := auc.AccountUseCase.GetAccountCPF(cpf)
+	id, err := auc.AccountUseCase.GetAccountCPF(cpf)
+
+	if err != nil {
+		return domain.ErrInsert, ""
+	}
+
 	now := domain.CreatedAt()
-	token := now + ":" + id.ID.String()
+	aux2 := 0
+	var idToken uuid.UUID
+	for aux2 == 0 {
+		idToken, _ = domain.Random()
+		searchToken, err := auc.SearchToken(idToken)
+		if err != nil {
+			return err, ""
+		}
+		if (searchToken != domain.Token{}) {
+			aux2 = 0
+		} else {
+			aux2 = 1
+		}
+	}
```

Comment:
hey @CMedrado , can't figure out what you're trying to do here either? Are u just checking if the generated uuid is unique? If so it's not necessary since its collision probability is extremely low